content/en/observability_pipelines/setup/datadog.md

```diff
@@ -39,6 +39,9 @@ You can generate both of these in [Observability Pipelines][3].
 
 ### Provider-specific requirements
 {{< tabs >}}
+{{% tab "Docker" %}}
+There are no provider-specific requirements for Docker.
```

Comment:
We should synchronize this and the Splunk guide, as they give different "provider-specific" instructions for Docker.

Reply:
Good catch! Forgot to update it here.